Adding option for adding default collapsed/expanded for Panel Layout. (#5770)
filamentphp/filament: packages/tables/docs/06-layout.md

]
```

> You can collapse a panel by default using the `collapsed()` method:

Reviewer: ```suggestion
You can also expand a panel by default using the `collapsed(false)` method:
```

]
```

You can expand a panel by default using the `collapsed(false)` method:

```php
use Filament\Tables\Columns\Layout\Panel;
use Filament\Tables\Columns\TextColumn;

Panel::make([
    Stack::make([
        TextColumn::make('email'),
        TextColumn::make('phone'),
    ]),

> ])
>     ->collapsible()
>     ->collapsed(false)

Reviewer: ```suggestion
])->collapsed(false)
```

]
```

> Also default collapse mode can be set for `Panel` Layout using `collapsed` method.

Reviewer: ```suggestion
You can collapse a panel by default using the `collapsed()` method:
```

```php
use Filament\Tables\Columns\Layout\Panel;
use Filament\Tables\Columns\TextColumn;

> [

Reviewer: Please remove the first example, we only need the `collapsed(false)` one
